FluentTc/Engine/BuildConfigurationRunner.cs

```diff
@@ -49,17 +63,48 @@ private static List<Property> GetProperties(Action<IBuildParameterValueBuilder>
             return buildParameterValueBuilder.GetParameters();
         }
 
-        private static string CreateTriggerBody(string buildConfigId, int? agentId, List<Property> properties = null)
+        private static string CreateTriggerBody(string buildConfigId, int? agentId, List<Property> properties = null, MoreOptionsHavingBuilder moreOptions = null)
         {
             var bodyBuilder = new StringBuilder();
-            bodyBuilder.Append(@"<build>").AppendLine()
-                .AppendFormat(@"<buildType id=""{0}""/>", buildConfigId).AppendLine();
+
+            if (moreOptions != null &&
+                moreOptions.TriggeringOptions.Personal == true)
+            {
+                bodyBuilder.Append(@"<build personal=""true"">").AppendLine();
+            }
+            else
+            {
+                bodyBuilder.Append(@"<build>").AppendLine();
+            }
+            
+            bodyBuilder.AppendFormat(@"<buildType id=""{0}""/>", buildConfigId).AppendLine();
 
             if (agentId.HasValue)
             {
                 bodyBuilder.AppendFormat(@"<agent id=""{0}""/>", agentId).AppendLine();
             }
 
+            if (moreOptions != null &&
+                moreOptions.GetComment() != null)
+            {
+                bodyBuilder.AppendFormat(@"<comment><text>{0}</text></comment>", moreOptions.GetComment()).AppendLine();
```

Comment:
What happens if comment contains some characters that might break the XML, like < > /?